src/database/gravity-db.c

```diff
@@ -102,6 +103,13 @@ static void gravity_check_ABP_format(void)
 	// Check if we have a valid ABP format
 	// We do this by checking the "abp_domains" property in the "info" table
 
+	// Return early if the database is not open
```

Comment:
Why did you add this code? 

`gravity_check_ABP_format()` is only executed once in the whole repo - as part of `gravityDB_open()` where we have checks before to make sure the database exists.

Reply:
You are absolutely right, I force-pushed this away now.